Source/Scene/Scene.js

```diff
@@ -539,6 +541,18 @@ define([
          */
         this.fog = new Fog();
 
+        this._sunCamera = new Camera(this);
```

Comment:
It is needed for now, but only because someone could change `Scene.shadowMap` to something else, like in the Sandcastle.